doc/index.rst

```diff
@@ -34,9 +34,18 @@ There are three different ways which you can use Cube Browser:
 Installation
 ____________
 
-Conda is great
+In the following pages you will find downloads of Jupyter notebooks containing Cube Browser operations.
+To be able to use these notebooks, you will need to configure your environment correctly.
+The most efficient and clean method to do this is to use a Conda environment.
+
+If you have never used Conda before, you will need to follow these `quick installation instructions <http://conda.pydata.org/docs/install/quick.html>`_ to get started.
+
+Once you have done this, open a terminal and type:
+
+``conda install -c scitools -c conda-forge -c scitools/label/dev cube_browser``
```

Comment:
I don't think this is detailed enough... If you just run this (assuming conda is available) then it will install it all into your root env, which may well not be preferable.

Alternatively:

```bash
conda create -n cb python
. activate cb
conda install -c scitools -c conda-forge -c scitools/label/dev cube_browser
```

Or (like what we had before):

```bash
conda create -n cb -c -c scitools -c conda-forge -c scitools/label/dev cube_browser
```

The first one has a lot of lines, the second one squashes an awful lot onto a single line...